doc/classes/RayCast.xml

```diff
@@ -140,6 +142,7 @@
 			If [code]true[/code], collisions will be reported. Default value: [code]false[/code].
 		</member>
 		<member name="exclude_parent" type="bool" setter="set_exclude_parent_body" getter="get_exclude_parent_body">
+			If [code]true[/code], collisions will be ignored for this RayCast's immediate parent. Default value: [code]true[/code].
```

Comment:
Please remove comma.

Reply:
Done. I removed the comma from the line above it also since it had the same sentence structure... for consistency.